css/main.scss

```diff
@@ -39,6 +39,9 @@ $base-hue: 150;
 @mixin for-desktop-only {
   @media (min-width: 1200px) and (max-width: 1799px) { @content; }
 }
+@mixin for-most-desktops {

```

Comment:
I think this must be causing the problem I'm seeing on my wide screen that the max it goes to is 3 images wide, but the 4th column gets popped to the bottom.